docs/apis/plugintypes/repository/_examples/full_list.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
This file is not valid PHP. It's actually a jsonc-like structure. Maybe reformat it as that?

Reply:
Totally, I will change it to jsonc file